src/Components/UserProfile/User.js

```diff
@@ -38,7 +38,7 @@ function User({ singleUser }) {
       {!singleUser && (
         <Navbar
           start={
-            <Link to="/" aria-label="Go back to Home">
+            <Link to="/search" aria-label="Go back to Home">
```

Comment:
You should have to change the label too! 
```suggestion
            <Link to="/search" aria-label="Go back to Search">
```

Reply:
Got it! Thanks for the feedback 👍